coralproject/talk
# [CORL 212] Implement CharCount Restriction feature

## What does this PR do?
- Implement CharCount Restriction feature on the Stream
- Removes "Powered by Coral" text [CORL-262]

## How do I test this PR?

- Set char count restriction and check on client (post, edit, reply)
- Update to a stricter char count restriction while client only knows about the outdated char count restriction. A server error would trigger the client to refresh the setting
